miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/PoolDto.java

```diff
@@ -208,4 +209,12 @@ public int getLongestIndex() {
   public void setLongestIndex(int longestIndex) {
     this.longestIndex = longestIndex;
   }
+
+  public boolean isHasLowQualityLibraries() {
```

Comment:
isHas?

Reply:
It's what Eclipse generated. :-( This bean naming convention makes me sad.